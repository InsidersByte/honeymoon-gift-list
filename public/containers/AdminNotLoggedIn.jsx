/* @flow */

import React from 'react';

type PropsType = {
    children: React$Element<any>,
};

const styles = {
    root: {
        height: '100%',
        display: 'flex',
        alignItems: 'center',
        justifyContent: 'center',
        paddingBottom: '8vh',
    },
    container: {
        margin: '0 5%',
    },
};

const AdminLoggedIn = ({ children }: PropsType) => (
    <div style={styles.root}>
        <div style={styles.container}>
            {children}
        </div>
    </div>
);

export default AdminLoggedIn;
