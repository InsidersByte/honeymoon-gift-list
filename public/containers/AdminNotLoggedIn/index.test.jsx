import React from 'react';
import { shallow } from 'enzyme';
import AdminNotLoggedIn from './';

describe('AdminNotLoggedIn', () => {
    it('should render correctly', () => {
        const wrapper = shallow(
            <AdminNotLoggedIn>
                <h1>Hello World</h1>
            </AdminNotLoggedIn>,
        );

        expect(wrapper).toMatchSnapshot();
    });
});
