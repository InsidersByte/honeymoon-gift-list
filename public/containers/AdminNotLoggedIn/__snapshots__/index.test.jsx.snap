// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AdminNotLoggedIn should render correctly 1`] = `
<div
  style={
    Object {
      "alignItems": "center",
      "display": "flex",
      "height": "100%",
      "justifyContent": "center",
      "paddingBottom": "8vh",
    }
  }
>
  <div
    style={
      Object {
        "margin": "0 5%",
      }
    }
  >
    <h1>
      Hello World
    </h1>
  </div>
</div>
`;
